modules/installation-guide/partials/proc_installing-the-che-operator.adoc

```diff
@@ -15,11 +19,10 @@
 
 . In the *{prod}* pop-up window, click the btn:[Install] button.
 
-. On the *Install Operator* screen, choose following options and validate:
+. On the *Install Operator* screen, specify the following options:
 +
 * *Installation mode*: *A specific {orch-namespace} on the cluster*
-* *Installed Namespace*: `{prod-short}`
-//(Operator recommended namespace: ProjectPRopenshift-workspaces) ?
+* *Installed Namespace*: *Pick an existing namespace* -> `{prod-short}`
```

Comment:
Just to make sure... shouldn't the value be `{prod-id-short}` here?

Reply:
It is not, see my commend above.